Src/Sarif.PatternMatcher.Sdk/Crc32.cs

```diff
@@ -7,6 +7,9 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk
 {
     public static class Crc32
     {
+        public static readonly uint[] Crc32CastagnoliTable = CreateCrcTable(Crc32CastagnoliPolynomial);
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

Why is this public? This public array of items is mutable, i.e., any caller can change index 0 this data and break it. That's not good.

This data should either not be public or we should be exposing an immutable array.